numpy/ma/core.py

```diff
@@ -5838,6 +5838,36 @@ def max(self, axis=None, out=None, fill_value=None, keepdims=np._NoValue):
         ma.maximum_fill_value
             Returns the maximum filling value for a given datatype.
 
+        Examples
+        --------
+        >>> import numpy.ma as ma
+        >>> x = [[-1., 2.5], [4., -2.], [3., 0.]]
+        >>> mask = [[0, 0], [1, 0], [1, 0]]
+        >>> masked_x = ma.masked_array(x, mask)
+        >>> masked_x
+        masked_array(
+          data=[[-1.0, 2.5],
+                [--, -2.0],
+                [--, 0.0]],
+          mask=[[False, False],
+                [ True, False],
+                [ True, False]],
+          fill_value=1e+20)
+        >>> ma.max(masked_x)
+          2.5
+        >>> ma.max(masked_x, axis=0)
+          masked_array(data=[-1.0, 2.5],
```

Comment:
Ooops, maybe I missed this one?  This and the result above are indented by 2 spaces now, but they should not be.
I guess I might have marked the wrong line (aligning the indentation of mask and fill_value rather than fixing that of the `masked_array` line itself.)

Reply:
Changed it now